vignettes/estimate_ascertainment.Rmd

```diff
@@ -197,17 +167,16 @@ onset_to_death_covid <- epidist_db(
 )
 ```
 
-To visualise this distribution, we evaluate it between 0 and 30 days, and plot
-the results over time. We do so using the following command:
+### Estimating the proportion of cases that have been ascertained
 
-```{r fig.cap = "Example plot of the appropriate delay distribution for the COVID-19 epidemic in the U.K.We plot the onset-to-death distribution we use throughout this example for COVID-19, reported in https://doi.org/10.3390/jcm9020538."}
-plot(onset_to_death_covid, day_range = seq_len(30))
-```
+We use the `estimate_reporting()` function within the _cfr_ package to calculate the time-varying CFR for the COVID-19 epidemic in the U.K.
+
+The function includes a `type` argument, which determines whether `estimate_static()` or `estimate_time_varying()` is used to estimate the delay-adjusted severity of the disease.
 
-## Estimating the proportion of cases that have been ascertained
+The `severity_baseline` argument in the `estimate_reporting()` determines the denominator in the resulting under-ascertainment calculation.
```

Comment:
I know it's mentioned above that this is a result of the ratio of the true baseline and the delay adjusted CFR, but for people who are reading this for the first time it might not be immediately obvious what denominator is referring to here, adding the formula above to spell it out would be helpful

Reply:
Thanks, replaced with

> The ascertainment rate is calculated as the disease severity calculated from the data, divided by the 'known' disease severity; this is expected to be known or assumed from our best knowledge of the pathology of the disease.
> 
> This known disease severity is passed to the `severity_baseline` argument in `estimate_reporting()`, and forms the denominator in the resulting under-ascertainment calculation.